openprescribing/frontend/management/commands/generate_measure_fixtures.py

```diff
@@ -0,0 +1,276 @@
+# This task generates the fixtures in frontend/tests/fixtures/functional-measures.json.
+# The generated data includes Practices and all their parent organisations,
+# Measures, MeasureValues, and MeasureGlobals.
```

Comment:
Perhaps mention that when run, a test environment is automagically selected, as it alters database state?

Reply:
Also mention that it sets up test fixtures in BigQuery as well as JSON fixtures locally

Reply:
I am worried a future dev might not notice that this command exists, and would start editing the fixtures directly. Hard to solve though. Perhaps a sentence in the testing section of the README; and a comment at the top of the measures test files?